### ISSUE TITLE
Upgrade from 8.1.5 to 8.2.0 failed

### ISSUE THREAD

**Issue by emansije** [espocrm/espocrm#3021]
**Describe the bug**
When I tried to upgrade from version 8.1.5 to version 8.2.0, it threw an error and broke the installation. My system is an updated Ubuntu 22.04.4 LTS server (jammy, x86_64 arch) with PHP 8.1 fpm running on Nginx version 1.18.0-6ubuntu14.4.

**To Reproduce (MANDATORY, DO NOT REMOVE)**
Explicit steps to reproduce the behavior:
1. `cd` into the EspoCRM installation directory
2. `sudo -u www-data php command.php upgrade`
3. `sudo systemctl restart php8.1-fpm.service`

The output of step 2 was the following:

```
Current version is 8.1.5.
EspoCRM will be upgraded to version 8.2.0 now. Enter [Y] to continue.
y
This may take a while. Do not close the terminal.
Downloading...
Upgrading...
  Initialization...
  Copying before upgrade files...
  Rebuilding...
  Before upgrade script execution...
  Rebuilding...
  Copying files...
  Rebuilding...
  Reverting...
Unknown error on shell_exec.
```

**Expected behavior**
An upgrade without errors, like it did in previous upgrades.

**EspoCRM version**
The installed version is 8.1.5 and was trying to upgrade to version 8.2.0.

**Comment by yurikuzn** [comment]
https://forum.espocrm.com/forum/installation-and-upgrade-help

**Comment by yurikuzn** [comment]
Most likely due custom repositories. See upgrade notes https://github.com/espocrm/espocrm/releases/tag/8.2.0.

**Comment by emansije** [comment]
You're absolutely right! I have an extension installed (Real Estate) that I had completely forgotten about. After reading the upgrade notes I went looking for extensions, and there it was! I upgraded it and after that the EspoCRM upgrade went just fine. Thank you!
